binr/r2pm/r2pm

```diff
@@ -84,7 +84,7 @@ TRAVIS_TYPE=XX
 TRAVIS_JOB=86948888
 IS_SYSPKG=0
 [ -z "$R2PM_USRDIR" ] && R2PM_USRDIR="${HOME}/.config/radare2/r2pm"
-R2PM_ETCD="${R2HOMEDIR}/radare2rc.d"
+R2PM_ETCD="${HOME}/.config/radare2/radare2rc.d"
```

Comment:
what about using r2 -H for this

Reply:
It is a fastfix. I think it is enough until @xarkes reworkes the r2pm.